dev/src/reference.ts

```diff
@@ -1615,7 +1615,7 @@ export class Query<T = firestore.DocumentData> implements firestore.Query<T> {
    * @param aggregates the aggregations to perform.
    * @return an `AggregateQuery` that performs the given aggregations.
    */
-  aggregate<T extends firestore.AggregateSpec>(
+  _aggregate<T extends firestore.AggregateSpec>(
```

Comment:
Explicitly change the visibility of ` _aggregate` to `private`, using the `private` keyword.

Reply:
Done